app/src/main/java/org/mozilla/focus/activity/TextActionActivity.java

```diff
@@ -38,9 +38,9 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         }
         final String searchUrl = SearchUtils.createSearchUrl(this, searchText);
 
-        final Intent searchIntent = new Intent(this, MainActivity.class);
+        final Intent searchIntent = new Intent(this, RocketLauncherActivity.class);
```

Comment:
How about `searchIntent.setClassName(this, AppConstants.LAUNCHER_ACTIVITY_ALIAS);`?